src/ServerScriptService/InstanceComponents/Misc/Character.lua

```diff
@@ -23,6 +23,7 @@ function Character.new(instance: Instance)
 
 	-- Add CameraLook tag to the character
 	CollectionService:AddTag(instance, "CameraLook")
+	CollectionService:AddTag(instance, "FallDamage")
```

Comment:
The fall damage tag doesn't make sense to be under the "CameraLook tag" comment block. I would recommend changing the comment to read "Add character tags" or something of the sort.

Reply:
``Add tags to the character`` might be better wording.